lib/session/inbound-call-session.js

```diff
@@ -33,13 +33,24 @@ class InboundCallSession extends CallSession {
   }
 
   _onCancel() {
+    this.rootSpan.setAttributes({'call.termination': 'caller abandoned'});
```

Comment:
Could we consider to remove this duplication.

Reply:
Sure. We can remove this.